src/validators/location.js

```diff
@@ -1,43 +1,123 @@
+import { validate } from 'validate.js'
+import usStates from 'constants/enums/usStates'
+import usTerritories from 'constants/enums/usTerritories'
+import militaryStatesEnum from 'constants/enums/militaryStates'
+
 import { api } from '../services/api'
 import Layouts from '../components/Form/Location/Layouts'
 import { isZipcodeState, zipcodes } from '../config'
 import { isDefined } from './helpers'
 
-export const isInternational = location => {
-  return !['United States', 'POSTOFFICE'].includes(
-    countryString(location.country || {})
-  )
-}
-
-export const countryString = country => {
-  if (country) {
+export const countryString = (country) => {
+  if (validate.isHash(country)) {
     if (isDefined(country.value)) {
       if (Array.isArray(country.value)) {
         return country.value[0]
       }
 
       return country.value
-    } else if (country.value === null) {
+    }
+
+    if (country.value === null) {
```

Comment:
Can we move the conditional check for `country.value === null` and `country.value === undefined` nested at the top of `if (validate.isHash(country)) {` and delete `if (isDefined(country.value)) {`.

Maybe something like this

```
export const countryString = (country) => {
  if (validate.isHash(country)) {
    if (country.value === null || country.value === undefined) {
      return country.value
    }

    if (Array.isArray(country.value)) {
      return country.value[0]
    }

    return country.value
  }
  return country
}
```

Reply:
👍 good call